examples/cstg-euid/html/index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>EUID Publisher Client-Side Integration Example using UID2 JavsScript SDK</title>
+    <link rel="stylesheet" type="text/css" href="/stylesheets/app.css" />
+    <link rel="shortcut icon" href="/images/favicon.png" />
+    <script defer src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+    <script defer src="{{ EUID_JS_SDK_URL }}"></script>
+    <script>
+      const clientSideIdentityOptions = {
+        subscriptionId: '{{ SUBSCRIPTION_ID }}',
+        serverPublicKey: '{{ SERVER_PUBLIC_KEY }}',
+      };
+
+      let callbackCounter = 0;
+
+      function updateGuiElements(state) {
+        $(`#targeted_advertising_ready`).text(__euid.getAdvertisingToken() ? 'yes' : 'no');
+        $(`#advertising_token`).text(String(__euid.getAdvertisingToken()));
+        $(`#login_required`).text(
+          __euid.isLoginRequired() || __euid.isLoginRequired() === undefined ? 'yes' : 'no'
+        );
+        $(`#update_counter`).text(callbackCounter);
+        $(`#identity_state`).text(String(JSON.stringify(state, null, 2)));
+
+        const euidLoginRequired = __euid.isLoginRequired();
+        if (euidLoginRequired) {
+          $('#login_form').show();
+        } else {
+          $('#login_form').hide();
+        }
+        if (euidLoginRequired) {
+          $('#logout_form').hide();
+        } else {
+          $('#logout_form').show();
+        }
+      }
+
+      function onEuidIdentityUpdated(eventType, payload) {
+        console.log('EUID Callback', payload);
+        if (
+          payload?.identity &&
+          (eventType === 'InitCompleted' || eventType === 'IdentityUpdated')
+        ) {
+          ++callbackCounter;
+        }
+        updateGuiElements(payload);
+      }
+
+      function onDocumentReady() {
+        $('#logout').click(() => {
+          __euid.disconnect();
+          updateGuiElements(undefined);
+        });
+
+        $('#login').click(async () => {
+          const email = $('#email').val();
+
+          try {
+            await __euid.setIdentityFromEmail(email, clientSideIdentityOptions);
+          } catch (e) {
+            console.error('setIdentityFromEmail failed', e);
+          }
+        });
+      }
+
+      window.__euid = window.__euid || {};
+      window.__euid.callbacks = window.__euid.callbacks || [];
+
+      window.__euid.callbacks.push(onEuidIdentityUpdated);
+      window.__euid.callbacks.push((eventType, payload) => {
+        if (eventType === 'SdkLoaded') {
+          window.__euid.init({
+            baseUrl: '{{ UID2_BASE_URL }}',
```

Comment:
You'll need to provide EUID values for this and the keys, but I don't think we need them to be provided by the server - the whole point of CSTG is that the server isn't needed :) so they can be hard-coded here.